frontend/src/app/modules/downloads/downloads-page/downloads.component.html

```diff
@@ -9,6 +9,8 @@ <h3 class="mat-h3">Downloads</h3>
 
     <div class="horizontal-separator"></div>
 
+    <img class="lines-background" src="/assets/lines.svg" alt="Lines Background" />
```

Comment:
can you make background more according to "blue" style?

Reply:
no, this is image